src/main/java/org/prebid/server/bidder/yieldmo/YieldmoMetaInfo.java

```diff
@@ -0,0 +1,28 @@
+package org.prebid.server.bidder.yieldmo;
+
+import org.prebid.server.bidder.MetaInfo;
+import org.prebid.server.proto.response.BidderInfo;
+
+import java.util.Collections;
+
+/**
+ * Defines Yieldmo meta info
+ */
+public class YieldmoMetaInfo implements MetaInfo {
+
+    private BidderInfo bidderInfo;
+
+    public YieldmoMetaInfo(boolean enabled, boolean pbsEnforcesGdpr) {
+        bidderInfo = BidderInfo.create(enabled, "progsupport@yieldmo.com",
+                Collections.emptyList(), Collections.singletonList("banner"),
+                null, 0, pbsEnforcesGdpr);
```

Comment:
Pls set the GDPR vendor id from https://vendorlist.consensu.org/vendorlist.json

Reply:
Thanks for the link - I wasn't aware of it and just took what they've had in GO, which was 0.